### PR TITLE
terraform: Fix resource formatting for IPv6 CIDRs

### PR DESCRIPTION
**Pull request by a8m**
Solved #5976

### CODE REVIEW

File: tests/integration/update_cluster/restrict_access/id_rsa.pub [hunk 1]
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAAAgQCtWu40XQo8dczLsCq0OWV+hxm9uV3WxeH9Kgh4sMzQxNtoU1pvW0XdjpkBesRKGoolfWeCLXWxpyQb1IaiMkKoz7MdhQ/6UKjMjP66aFWWp3pwD0uj0HuJ7tq4gKHKRYGTaZIRWpzUiANBrjugVgA+Sd7E/mYwc/DMXkIyRZbvhQ==
```
Comment (chrisz100) [inline tests/integration/update_cluster/restrict_access/id_rsa.pub]: Who owns this key? What is it for?

Comment (a8m) [reply tests/integration/update_cluster/restrict_access/id_rsa.pub]: This is the same key that is used by all integration tests. For example, the [shared vpc test]. If you remove it, the test will fail with the following error:
```
--- FAIL: TestRestrictAccess (0.01s)
    integration_test.go:238: error running "in-v1alpha2.yaml" create: error reading SSH public key
    ....
```


[shared vpc test]: https://github.com/kubernetes/kops/blob/master/tests/integration/update_cluster/shared_vpc/id_rsa.pub

**Comment by a8m** [comment]
One of the tests was failing after my changes. I fixed it.
Running `terraform validate` on this file (before my changes) was failing with the following error:
```console
Error: aws_autoscaling_attachment.exttg-aws:my-tg--0123456789abcdef-master-us-test-1a: 
exttg-aws:my-tg--0123456789abcdef-master-us-test-1a: resource name can only contain
letters, numbers, dashes, and underscores.
```

**Comment by a8m** [comment]
One of the tests was failing after my changes. I fixed it.
Running `terraform validate` on this file (before my changes) was failing with the following error:
```console
Error: aws_autoscaling_attachment.exttg-aws:my-tg--0123456789abcdef-master-us-test-1a: 
exttg-aws:my-tg--0123456789abcdef-master-us-test-1a: resource name can only contain
letters, numbers, dashes, and underscores.
```
The problem was that `kops` was generating invalid terraform files. My suggestions is to run `terraform validate` as part of the integration tests. But this can be made in a different PR.

File: tests/integration/update_cluster/restrict_access/id_rsa.pub (hunk 1 above)
Comment (justinsb) [reply tests/integration/update_cluster/restrict_access/id_rsa.pub]: Yeah, it's a little annoying, but it's just a one-off key I generated.  It isn't used for anything but the tests.

File: tests/integration/update_cluster/restrict_access/in-v1alpha2.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  kubernetesVersion: v1.8.0
+  masterInternalName: api.internal.restrictaccess.example.com
+  masterPublicName: api.restrictaccess.example.com
+  networkCIDR: 172.20.0.0/16
+  networking:
+    kubenet: {}
+  nonMasqueradeCIDR: 100.64.0.0/10
+  sshAccess:
+  - 1.1.1.1/0
+  - 2001:0:85a3::/40
```
Comment (justinsb) [inline tests/integration/update_cluster/restrict_access/in-v1alpha2.yaml]: You probably could have just added this to `complex`, which is a sort of grab-bag for random things.  But this works :-)

**Comment by justinsb** [comment]
Thanks @a8m 

/approve

**Pull request merged**